src/main.py

```diff
@@ -12,85 +12,163 @@
 from selenium.common.exceptions import NoSuchElementException
 from webdriver_manager.chrome import ChromeDriverManager
 from selenium.webdriver.common.by import By
+from selenium.webdriver.chrome.options import Options
 
 # Load in the credentials from .env
 load_dotenv()
 
-# Get the Chromium web driver
-driver = webdriver.Chrome(ChromeDriverManager().install())
-
 # Set the email and password variables from the environment variables
 email = os.getenv('EMAIL')
 password = os.getenv('PASSWORD')
+token = os.getenv('TOKEN')
+newpassword = os.getenv('NEWPASSWORD')
 
-def main():
-	#Blocking cloudflare, sentry.io and discord science URLS (analytics and monitoring URLS)
+def totp(gen):
+	# Get the Chromium web driver
+	driver = webdriver.Chrome(ChromeDriverManager().install())

```

Comment:
It is potentially inefficient to create the WebDriver inside the main function.

Reply:
As mentioned first time do anything thing with python but I did this as to allow for the user input for type of code and if they're trying to do a password reset.